docker/test_env/eth2_common_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build eth_env_tests
+// +build eth_env_tests
```

Comment:
I think we only need one of these. The top one is for go 1.17 and above while the lower one is for 1.16 and below. We are well beyond 1.17 at this point and I don't think we want to go back and support older versions that far back.

Reply:
for some reason some plugin in my VCS adds `go:build` on save even if I remove it... can we keep it? I think no harm done?

Reply:
It's for compatibility reasons, no harm done.